scripts/dasharo-deploy

```diff
@@ -316,17 +577,17 @@ check_vboot_keys() {
     # No FMAP flashing? Also skip
     grep -q "\--fmap" <<< "$FLASHROM_ADD_OPT_UPDATE" || return
 
-    BINARY_KEYS=$(CBFSTOOL=$(which cbfstool) futility show $BIOS_UPDATE_FILE| grep -i 'key sha1sum')
+    BINARY_KEYS=$(CBFSTOOL="$CBFSTOOL" $FUTILITY dump_vboot_keys show $BIOS_UPDATE_FILE| grep -i 'key sha1sum')
 
     if [ $BOARD_HAS_FD_REGION -eq 0 ]; then
       FLASHROM_ADD_OPT_READ=""
     else
       FLASHROM_ADD_OPT_READ="--ifd -i bios"
     fi
     echo "Checking vboot keys."
-    $FLASHROM -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} ${FLASHROM_ADD_OPT_READ} -r /tmp/bios.bin > /dev/null 2>/dev/null
-    if [ $? -eq 0 ] && [ -f "/tmp/bios.bin" ]; then
-      FLASH_KEYS=$(CBFSTOOL=$(which cbfstool) futility show /tmp/bios.bin | grep -i 'key sha1sum')
+    $FLASHROM read_firm_mock -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} ${FLASHROM_ADD_OPT_READ} -r $BIOS_DUMP_FILE > /dev/null 2>/dev/null
+    if [ $? -eq 0 ] && [ -f $BIOS_DUMP_FILE ]; then
+      FLASH_KEYS=$(CBFSTOOL="$CBFSTOOL" $FUTILITY dump_vboot_keys show $BIOS_DUMP_FILE | grep -i 'key sha1sum')
```

Comment:
Comment above

Reply:
So you did change this from `CBFSTOOL="$CBFSTOOL"` to `CBFSTOOL="which cbfstool"`, previous commend only said about fix for typo.
It's very unintuitive compared to how it's normally used. If it worked as it was previously then maybe you should keep it otherwise it'll make code very hard to read or maintain.